### ISSUE TITLE
openAPI with Keycloak REST API does not work

### ISSUE THREAD

**Issue by timcouchoud** [wundergraph/wundergraph#559]
### Bug description

I am trying to fetch all users from Keycloak through openAPI data source from KEycloak REST API.
--> wundergraph app crashes by starting because of following error:

```
023-01-18T19:08:39+01:00 FATAL: Couldn't configure your WunderNode: TypeError: Reduce of empty array with no initial value {"component":"@wundergraph/sdk"}
...
2023-01-18T19:08:39+01:00 ERROR scriptrunner/scriptrunner.go:154 Script runner exited with non-zero exit code {"component": "@wundergraph/wunderctl", "runnerName": "config-runner", "exit": 1, "startTs": 1674065317388697093, "stopTs": 1674065319267649929, "complete": true}
...
2023-01-18T19:08:39+01:00 ERROR apihandler/apihandler.go:411 EndpointUnavailableHandler {"component": "@wundergraph/node", "Operation": "Users", "Endpoint": "/operations/Users", "Help": "This operation is invalid. Please, check the logs"}
...
/home/XXXXXXXX/node_modules/@wundergraph/sdk/dist/v2openapi/index.js:1020
                .reduce((prev, current) => {
                 ^

TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at RESTApiBuilder.resolveFieldName (/home/XXXXXXXX/weesdom/node_modules/@wundergraph/sdk/dist/v2openapi/index.js:1020:18)
    at RESTApiBuilder.traversePath (/home/XXXXXXX/weesdom/node_modules/@wundergraph/sdk/dist/v2openapi/index.js:92:57)
    at /home/XXXXXXX/weesdom/node_modules/@wundergraph/sdk/dist/v2openapi/index.js:49:26
    at Array.forEach (<anonymous>)
    at RESTApiBuilder.build (/home/XXXXXXXX/weesdom/node_modules/@wundergraph/sdk/dist/v2openapi/index.js:43:42)
    at openApiSpecificationToRESTApiObject (/home/XXXXXXX/weesdom/node_modules/@wundergraph/sdk/dist/v2openapi/index.js:30:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async generator (/home/XXXXXXX/weesdom/node_modules/@wundergraph/sdk/dist/definition/index.js:315:20)
    at async Promise.all (index 1)
```

### Expected behavior

wundergraph app starts

### WunderGraph information

wundergraph.config.ts file is:
```
// KEYCLOAK data source
const keycloak = introspect.openApi({
	apiNamespace: 'keycloak',
	baseURL: "http://localhost:8080/admin/realms/demo/",
	source: {
		kind: 'file',
		filePath: './keycloak_openAPI.json',
	},
  })
```
keycloak_openAPI.json is a copy of https://github.com/ccouzens/keycloak-openapi/blob/main/keycloak/19.0.0.json

users.graphql file is:
```
query Users {
  users {
    id
    username
  }
}
```

### Environment & setup

- OS: 
- Go version: 
- Database: 
- Node.js version: 

### WunderCtl Version

    "@wundergraph/react-query": "^0.2.4",
    "@wundergraph/sdk": "^0.130.1",

**Comment by Slickstef11** [comment]
Thank you for reporting @timcouchoud Has this been fixed? I think I saw it in the Discord.

**Comment by fiam** [comment]
Fixed in #566
